src/Node/FS/Async.js

```diff
@@ -4,7 +4,21 @@
 
 // module Node.FS.Async
 
-exports.fs = require('fs');
+var fs = require('fs');
+exports.fs = fs
+
+exports.create = fs.open;
+exports.readSeq = fs.read;
```

Comment:
Looks like `exports.readSeq` is assigned twice - should one of these assignments be removed? Same for `writeSeq`.